tests/gitlab/README.md

```diff
@@ -7,6 +7,8 @@
 
                                https://mfem.org
 
+FIXME: this file needs to be updated
```

Comment:
Another FIXME: we probably should update this file as well.

Reply:
Proposed update